zipline/_protocol.pyx

```diff
@@ -459,7 +466,25 @@ cdef class BarData:
                         frequency,
                         field
                     )[assets] for field in fields
-                })
+                }
+
+                if self._adjust_minutes:
+                    adjs = {
+                        field: self.data_portal.get_adjustments(
+                            assets,
+                            field,
+                            self._get_current_minute(),
+                            self.simulation_dt_func()
+                        )[0] for field in fields
```

Comment:
data_portal.get_adjustments returns a list of adjustments for the list of assets that are passed in. In this case, since `assets` is just one asset, it gets coerced into a list, so the return value is a list of len 1